jest.config.unit.js

```
@@ -0,0 +1,60 @@
+  // Only match unit test files under src/
+  testMatch: ['<rootDir>/src/**/*.test.{ts,tsx}'],
+
+  // Ignore E2E tests
+  testPathIgnorePatterns: ['<rootDir>/e2e/', '<rootDir>/node_modules/'],
+
+  // Transform TypeScript
+  transform: {
+    '^.+\\.(ts|tsx)$': [
+      'ts-jest',
+      {
+        tsconfig: '<rootDir>/tsconfig.json',
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)